Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -317,6 +317,9 @@ final class CacheAdvanceTests: XCTestCase {
             let secondCacheMessages = try secondCache.messages()
```

Comment:
I'm pretty sure this test had a bug before since we were overwriting the cache file when creating a second one.
I printed out the result of `secondCacheMessages` and it was only one message.
Ideally we can rework this test so it will fail if we make the same mistake again, but I didn't have time to do so.

Reply:
good find!

I think the easiest solve here may be to assert that the second cache isn’t empty after creating it.

Reply:
Good idea https://github.com/dfed/CacheAdvance/pull/45/commits/8f18f894a84c39a90f81e6a36d95e0e72d91af98